### ISSUE TITLE
La magia de la terminal tambien se puede tener en CMD

### ISSUE THREAD

**Issue by JavierCorralLizarraga** [LornartheBreton/linux_workshop#4]
Existe Doskey y chocolatey para emular funcionalidades de otras terminales, si bien es pesado reaprender los comandos, no se propone que todo se haga desde esta terminal, pues existe wsl y desde ahi deberian hacerse las cosas bien, pero un poco de conocimiento de CMD y powershell no le estorba a nadie

**Comment by LornartheBreton** [comment]
Agreed, pero el punto es indoctrinarlos ideológicamente en el Linux-open source mindset.

Hacerlo con WSL/Powershell/Chocolatery podría ser contraproducente.
